### ISSUE TITLE
Upgrade Requirements so it can use Coreutils greater than 4.1

### ISSUE THREAD

**Issue by mwidjaja1** [IBM/jupyterlab-s3-browser#26]
I'd like to get version 0.6.0 because I'm running into #15 but am unable to because of this version mismatch. for coreutils.

package.json currently requests Coreutil's version to be `"@jupyterlab/coreutils": "~4.1"`. This seems off because an older version, version 0.4.1, actually has it at ` "@jupyterlab/coreutils": "^4.0.0"`. This means that _newer_ versions of Coreutils is actually directed to use an older version of jupyterlab-s3-browser instead of the latest one.

This seems like a typo but is this actually intentional by design? And if it isn't, can it just be swapped to be anything greater than 4.1 instead?

Thanks!

**Comment by reevejd** [comment]
Thanks for opening the issue. It should be fixed in version 0.6.2, can you confirm?

**Comment by mwidjaja1** [comment]
I'd be glad to @reevejd but I'll be honest, I'm a little of a Jupyter newb so I'm not sure how to test version 0.6.2. What should I do to grab that version?

**Comment by reevejd** [comment]
Running `jupyter labextension install jupyterlab-s3-browser` should get the latest version (now 0.6.2) for you. Afterwards you can verify by running `jupyter labextension list`.

**Comment by mwidjaja1** [comment]
Hmm I'm running into a bunch of incompatibility errors now. Unfortunately I can't easily paste the error message here due to the computer's location, but some things of note

* `fsevents@2.1.3` is incompatible with 'linux', and because its optional, was excluded from installation.
* `coreutils@1.5.3` has unmet dependency on `crypto@1.0.1`
* `ipyvolume > tslint@5.20.1` has unmet dependency on `typescript` & `@types/three@*`
* Can't resolve `base/js/dialog` from qgrid2/src.
* Multiple module names that differ in casing -- `jupyter/lab/staging/node_modules/moment/locale/en-SG.js` vs `en-sg.js`
* Command failed in `jupyter/lab/staging/node_modules/.bin/webpack --config webpack.prod.minimize.config.js` and JupyterLab failed to build.

**Comment by reevejd** [comment]
Can you tell me a bit more about how you're trying to use this extension and the commands you're running to get the above errors?

**Comment by mwidjaja1** [comment]
Hey sorry for taking a while to get back! I was redirected onto another fire so I couldn't poke into this too much (still couldn't poke around too much).

But I started on an AWS Amazon-Linux machine with version 0.4 of this package and tried running the command you gave me to upgrade. While running that command, I got the 'summarized' error messages I pasted before.

Is this what you meant or is there anything else that would assist you? I can also try again but it'd be after I put some more fires out unfortunately.

Thanks!

**Comment by reevejd** [comment]
I'll investigate and get back to you

**Comment by mwidjaja1** [comment]
Hey, I greatly apologize for my long delay in getting back with you. It's been fire after fire and I never had the time to check back on this.

So last time, I had the old version of this plugin installed. I ran the install command you gave me and got that horrible traceback. Today I uninstalled it completely, rebuilt JupyterLab, and then reinstalled it using the same command you gave me. This time it worked, with no traceback issues at all.

I don't know if something weird happens when users upgrade, but it looks like from a fresh install, everything is fine. This may not be the plugin's fault either -- I know my corporation does strange things with our AWS Box, so it's very possible that there may have been an internet glitch or whatnot during the first upgrade. 

Don't know if this helps but I'm happy now, so feel free to close this issue unless you wanted to investigate the upgrade case more. Thanks so much!
